### PR TITLE
[addons] add few missing functions to listitem interface

### PR DESCRIPTION
**Pull request by AlwinEsch**
## Description
This add functions where required on new addon interface to have
same functionality as on old system.

## Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h [hunk 1]
```diff
@@ -91,20 +91,18 @@ namespace gui
...
       const std::string& label = "",
       const std::string& label2 = "",
       const std::string& iconImage = "",
-      const std::string& thumbnailImage = "",
       const std::string& path = "")
       : CAddonGUIControlBase(nullptr)
     {
       m_controlHandle = m_interface->kodi_gui->listItem->create(m_interface->kodiBase, label.c_str(),
                                                                 label2.c_str(), iconImage.c_str(),
-                                                                thumbnailImage.c_str(), path.c_str());
```
Comment (ksooo) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h]: just a question: why have you removed all thumbnail support from listitem?

**Review by ksooo — APPROVED** [review]
Looking good

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h (hunk 1 above)
Comment (AlwinEsch) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h]: Is nowhere used and on Kodi itself changed to a `SetArt(...)` value

**Comment by AlwinEsch** [comment]
jenkins build this please

**Comment by AchimTuran** [comment]
Just also small minors and is again up to you to change them.

File: xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h (hunk 1 above)
Comment (ksooo) [inline xbmc/addons/kodi-addon-dev-kit/include/kodi/gui/ListItem.h]: okay

**Comment by ksooo** [comment]
jenkins build and merge

**Pull request merged**